src/teraranger_evo.cpp

```diff
@@ -0,0 +1,215 @@
+#include <ros/console.h>
+#include <string>
+
+#include <teraranger_hub/RangeArray.h>
```

Comment:
This is weird because we solve this dependency problem a long time ago. I will take look anyway